target/linux/ramips/mt76x8/base-files/etc/board.d/02_network

```diff
@@ -184,6 +185,11 @@ ramips_setup_macs()
 	wiznet,wizfi630s)
 		wan_mac=$(macaddr_add "$(mtd_get_mac_binary factory 0x28)" 1)
 		;;
+	wavlink,wl-wn577a2)
+		wan_mac=$(mtd_get_mac_binary factory 0x2e)
+		lan_mac=$(mtd_get_mac_binary factory 0x28)
+		label_mac=$(mtd_get_mac_binary factory 0x4)
```

Comment:
Can you provide an overview of addresses and increments?
Is the address in 0x4 found in another location as well?

Reply:
Looking at the hexdump of the 'factory' partition I identified the following MAC addresses. 
2.4GHz and 5GHz MACs are actually printed on a label on the device.

0x4   80:3f:5d::xx:xx:xx (2.4GHz)
0x2e   80:3f:5d::xx:xx:xx (LAN)
0x28   80:3f:5d::xx:xx:xx (WAN)
0x8004 80:3f:5d:xx:xx:xx (5GHz)

Reply:
Well, thanks, but what I need is the last byte (to see the increments).
What I'm actually looking for is whether 0x4 and 0x28 have the same address by coincidence, so we can move label MAC address to DTS.

Reply:
No, 0x4 and 0x28 have different values:
0x8004 80:3f:5d:xx:xx:c2 (5GHz)
0x04   80:3f:5d:xx:x:c1 (2.4GHz)
0x2e   80:3f:5d:xx:xx:c0 (LAN)
0x28   80:3f:5d:xx:xx:bf (WAN)

Reply:
Fine, then the statement here is correct.